client/my-sites/hosting/staging-site-card/card-content/staging-sync-card.tsx

```diff
@@ -31,34 +31,39 @@ const synchronizationOptions: CheckboxOptionItem[] = [
 	{
 		name: 'themes',
 		label: translate( 'Themes' ),
+		subTitle: translate( 'Everything in the themes directory.' ),
 		checked: false,
 		isDangerous: false,
 	},
 	{
 		name: 'plugins',
 		label: translate( 'Plugins' ),
+		subTitle: translate( 'Everything in the plugins directory.' ),
 		checked: false,
 		isDangerous: false,
 	},
 	{
 		name: 'uploads',
 		label: translate( 'Media Uploads' ),
+		subTitle: translate(
+			'Everything in the media library. You must also select ‘Site database‘ for restored media uploads to appear.'
+		),
 		checked: false,
 		isDangerous: false,
 	},
 	{
 		name: 'contents',
 		label: translate( 'wp-content Directory' ),
 		subTitle: translate(
-			'Everything in the wp-content directory, excluding themes, plugins, and uploads'
+			'Everything in the wp-content directory, excluding themes, plugins, and uploads.'
 		),
 		checked: false,
 		isDangerous: false,
 	},
 	{
 		name: 'roots',
 		label: translate( 'Web Root' ),
-		subTitle: translate( 'Everything in the WordPress root, including any non WordPress files' ),
+		subTitle: translate( 'Everything in the WordPress root, including any non WordPress files.' ),
```

Comment:
```suggestion
		subTitle: translate( 'All files in the root directory, including any non WordPress files.' ),
```

Does this exclude the wp-content directory?

Reply:
Yes, it excludes `wp-content/`.